userspace/libscap/compat/bpf.h

```diff
@@ -9,11 +9,12 @@
 #define _UAPI__LINUX_BPF_H__
 
 #include <linux/types.h>
-#include "bpf_common.h"
```

Comment:
I guess this mirrors bpf.h from the kernel tree? If so, which version?

Reply:
This is at 71d19214776e61b33da48f7c1b46e522c7f78221 commit (https://github.com/torvalds/linux/blob/71d19214776e61b33da48f7c1b46e522c7f78221/include/uapi/linux/bpf.h), the one that introduced `bpf_ktime_get_boot_ns`.  
Should i mention it?
Btw note that you reviewed an old version of the code (i added back the include for "bpf_common.h").